packages/samples/headless-ssr/src/components/search-page.tsx

```diff
@@ -6,16 +6,17 @@ import {
   hydrateInitialState,
 } from '@/src/common/engine';
 import {useEffect, useState} from 'react';
+import {HydrationMetadata} from './hydration-metadata';
 import {ResultList} from './result-list';
 
 export default function SearchPage({
   initialState,
 }: {
   initialState: SearchSSRState;
 }) {
-  const [hydrationResult, setHydrationResult] = useState<SearchCSRState | null>(
-    null
-  );
+  const [hydrationResult, setHydrationResult] = useState<
```

Comment:
would `hydration` -> `CSR` make it more readable here (and in other places as well)?